From 6fb9d5df14b3b0e21db458fd5fcd10691bb9fd0e Mon Sep 17 00:00:00 2001
From: Vasant Hegde <hegdevasant@linux.vnet.ibm.com>
Date: Tue, 11 Mar 2014 11:07:51 +0000
Subject: [PATCH 26/72] powerpc: Move VPD related tool under common code

VPD related tools (lsvpd, lscfg, lsmcode) is supported on both
pSeries and PowerNV platform. Hence moving these commands under
common code.

Also added support to grab vpd database (/var/lib/lsvpd).

Signed-off-by: Vasant Hegde <hegdevasant@linux.vnet.ibm.com>
---
 sos/plugins/powerpc.py |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)
---
 sos/plugins/powerpc.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/sos/plugins/powerpc.py b/sos/plugins/powerpc.py
index dfaacb9..8fd6b4f 100644
--- a/sos/plugins/powerpc.py
+++ b/sos/plugins/powerpc.py
@@ -52,14 +52,15 @@ class PowerPC(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
             self.add_cmd_output("ppc64_cpu --run-mode")
             self.add_cmd_output("ppc64_cpu --frequency")
             self.add_cmd_output("ppc64_cpu --dscr")
+            self.add_cmd_output("lscfg -vp")
+            self.add_cmd_output("lsmcode -A")
+            self.add_cmd_output("lsvpd --debug")
+            self.add_copy_spec("/var/lib/lsvpd/")
 
         if ispSeries:
             self.add_copy_spec("/proc/ppc64/lparcfg")
             self.add_copy_spec("/proc/ppc64/eeh")
             self.add_copy_spec("/proc/ppc64/systemcfg")
-            self.add_cmd_output("lscfg -vp")
-            self.add_cmd_output("lsmcode -A")
-            self.add_cmd_output("lsvpd --debug")
             self.add_cmd_output("lsvio -des")
             self.add_cmd_output("servicelog --dump")
             self.add_cmd_output("servicelog_notify --list")
-- 
1.9.3

